docs/Coiote_IoT_DM/Coiote_videos/How_to_perform_secure_Over-the-Air_updates.md

```diff
@@ -0,0 +1,16 @@
+# How to perform secure Over-the-Air (OTA) updates? (nRF 9160)
+
+The LwM2M standard makes Over-the-Air (OTA) updates straightforward, reducing the complexity of implementation and management. What's crucial is that it enhances security through authentication and encryption.
+
+To demonstrate the potential of a single update, we use an IoT device with nRF 9160 DK integrated with Anjay IoT SDK—a setup that enables LwM2M Connectivity.
```

Comment:
```suggestion
To demonstrate the potential of a single update, we will use an IoT device with nRF9160 DK integrated with Anjay IoT SDK— a setup that enables LwM2M Connectivity.
```

Also, what does `an IoT device with nRF9160 DK`? The board acts as an IoT Device itself so it doesn't make any sense for me. But maybe I am wrong...

Reply:
it's only to highlight the device